src/pages/messaging/customer-chat-api/v3.1/rtm-pushes/index.mdx

```diff
@@ -531,6 +531,7 @@ Informs that a Customer was disconnected. The payload contains the reason of Cus
 
 | Type                                 | Notes                                                                                                                                |
 | ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------ |
+| `access_token_revoked`               | Access token has been revoked.                                                                                                       |
```

Comment:
Looking at the https://github.com/livechat/customer-api/blob/API-9397-disconnect-on-access_token-revoked/connection/conns.go#L158 you send `expired` for older versions, not `revoked` 🤔

Reply:
😅 right, i just copied revoked in every version